src/oneview.js

```diff
@@ -29,31 +31,34 @@ import configLoader from './config-loader';
 const main = (robot) => {
   // load OneView configuration data from external file
   let oneview_config = configLoader(robot);
+  if(oneview_config == null) {
+    return;
```

Comment:
Should we log something here about not starting because the config file is empty?